### DISCUSSION TITLE
Everything disappears on WSL2 WIN10

### DISCUSSION THREAD

**Discussion by taitep** [JanDeDobbeleer/oh-my-posh#3076] [Troubleshoot] [answered]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I was setting up my terminals on my windows 10 pc, and installing oh my posh. After installing it on WSL (ubuntu) the prompt disappeared and the terminal did not respond to anything. Looked like a terminal application that just cleared the terminal and left echo on.
![bug_report_ohmyposh](https://user-images.githubusercontent.com/75326724/201607653-fb43301c-3f20-43cd-ba17-1c865ca8bdb1.png)

### Theme

Just installed it, so none.

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
Cannot run that. Bash does not listen to commands.
```

**Comment by JanDeDobbeleer** [comment]
@Taitep wait, what did you configure in your `.bashrc`? Pretty certain this is either a misconfiguration or something else causing this.

**Comment by taitep** [comment]
I added theese lines:
```
eval "$(oh-my-posh init bash)"
exec bash
```

**Reply by JanDeDobbeleer** [reply, marked as answer]
@Taitep `exec bash` isn't needed in there.

**Reply by taitep** [reply]
So i just need to remove that?

**Reply by JanDeDobbeleer** [reply]
@Taitep yes

**Reply by taitep** [reply]
thx
